packages/react-server-test-pages/gulpfile.babel.js

```diff
@@ -0,0 +1,22 @@
+const eslint = require("gulp-eslint");
+const gulp = require("gulp");
+const babel = require("gulp-babel");
+const logging = require("react-server-gulp-module-tagger");
+
+gulp.task("travis-ci", ["build", "eslint"]);
+
+gulp.task("build", () => gulp.src("src/**/*.js")
+	.pipe(logging())
+	.pipe(babel())
+	.pipe(gulp.dest("lib/"))
+);
```

Comment:
So, it's worth noting that IIRC that `react-server-cli` [will do babel transformation by itself](https://github.com/redfin/react-server/blob/master/packages/react-server-cli/bin/react-server-cli), so this build step is (I think?) superfluous. But maybe you want this for consistency with the other packages, so I guess it's ok.

The upside for allowing `react-server-cli` to do the babel compile is that I think you could just add an entry for `scripts.start` that was `react-server-cli`, and then the install instructions would just be `npm start`. Also, watching/hot loading works by default. Thoughts?